### PR TITLE
Fix : php 8.1 warnings

### PR DESCRIPTION
**Pull request by Hystepik**
Fix php 8.1 warnings

### CODE REVIEW

File: htdocs/comm/propal/list.php [hunk 1]
```diff
@@ -1911,7 +1915,11 @@
 			if (!$i) {
 				$totalarray['pos'][$totalarray['nbfield']] = 'p.total_tva';
 			}
-			$totalarray['val']['p.total_tva'] += $obj->total_tva;
+			if (empty($totalarray['val']['p.total_tva'])) {
```
Comment (frederic34) [inline htdocs/comm/propal/list.php:1918]: you can define `$totalarray['val']['p.total_tva'] = 0` at line 1589 and also the other

File: htdocs/compta/tva/class/tva.class.php [hunk 2]
```diff
@@ -345,8 +345,8 @@ public function fetch($id, $user = null)
 				$this->fk_user_creat = $obj->fk_user_creat;
 				$this->fk_user_modif = $obj->fk_user_modif;
 				$this->fk_account = $obj->fk_account;
-				$this->fk_type = $obj->fk_type;
-				$this->rappro  = $obj->rappro;
+				$this->fk_type = !empty($obj->fk_type) ? $obj->fk_type : "";
+				$this->rappro  = !empty($obj->fk_type) ? $obj->rappro : "";
```
Comment (frederic34) [inline htdocs/compta/tva/class/tva.class.php:349]: typo here

[the remark this answers is not in the document]
Comment (eldy) [reply htdocs/compta/tva/class/tva.class.php:349]: $obj->rappro ?? ""  is still a very obscur syntax.
What if value is '0' and 0 ?
For readability using old syntax looks better.

Comment (eldy) [reply htdocs/compta/tva/class/tva.class.php]: > typo here

Thanks

**Pull request merged**